src/server/config/env/runTime.js

```diff
@@ -78,11 +88,49 @@ const runTime = [
     },
     defaultValue: () => 3005,
   },
-  // holocron config, the modules and versions to use
+  {
+    name: 'HTTP_ONE_APP_DEV_CDN_PORT',
+    normalize: (input) => {
+      if (input) {
+        const parsed = parseInt(input, 10);
+        // make sure the parsed value is the same value as input
+        // input may be a string or a number, we don't want === in this case, just ==
+        if (Number.isNaN(parsed) || parsed != input) { // eslint-disable-line eqeqeq
+          throw new Error(`env var HTTP_ONE_APP_DEV_CDN_PORT needs to be a valid integer, given "${input}"`);
+        } else {
+          return parsed;
+        }
+      }
+      return undefined;
```

Comment:
`undefined` is coerced to a string in `process.env` as you found
does env-config-utils preprocessEnvVar.js support optional configuration?
(might need to add that? ex:
```javascript
// https://github.com/americanexpress/env-config-utils/blob/master/src/preprocessEnvVar.js#L74
// from 
process.env[name] = value;
// to
if (value === undefined) {
  delete process.env[name];
} else {
  process.env[name] = value;
}
```
but then, perhaps `normalize` should just return the original input instead? maybe we should open an issue in that repo to discuss the feature requirements and implications in detail?)

Reply:
This should be done in a separate effort since it involves changes to another library that are out of scope for this PR.

Reply:
My only concern after the latest changes are having `process.env.HTTP_ONE_APP_DEV_CDN_PORT === 'undefined'`, but perhaps that's fine for now as something attempting to start the dev CDN server in production mode should fail anyway.